packages/core/addon/components/visualization-config/line-chart.js

```diff
@@ -18,6 +18,11 @@ import layout from '../../templates/components/visualization-config/line-chart';
 export default Component.extend({
   layout,
 
+  init() {
+    this._super(...arguments);
+    this.curveOptions = ['line', 'spline', 'step'];
```

Comment:
Can we capitalize the first letter in these options?

Reply:
Done with helper in template